kroxylicious-systemtests/src/main/java/io/kroxylicious/systemtests/installation/kroxylicious/Kroxy.java

```diff
@@ -0,0 +1,84 @@
+/*
+ * Copyright Kroxylicious Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+package io.kroxylicious.systemtests.installation.kroxylicious;
+
+import java.io.IOException;
+import java.util.List;
+import java.util.Objects;
+
+import org.junit.jupiter.api.TestInfo;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import io.fabric8.kubernetes.api.model.Pod;
+
+import io.kroxylicious.systemtests.Constants;
+import io.kroxylicious.systemtests.Environment;
+import io.kroxylicious.systemtests.resources.manager.ResourceManager;
+import io.kroxylicious.systemtests.templates.kroxylicious.KroxyConfigTemplates;
+import io.kroxylicious.systemtests.templates.kroxylicious.KroxyDeploymentTemplates;
+import io.kroxylicious.systemtests.templates.kroxylicious.KroxyServiceTemplates;
+import io.kroxylicious.systemtests.utils.DeploymentUtils;
+
+import static io.kroxylicious.systemtests.k8s.KubeClusterResource.kubeClient;
+
+/**
+ * The type Kroxy.
+ */
+public class Kroxy {
+    private static final Logger LOGGER = LoggerFactory.getLogger(Kroxy.class);
+    private final String deploymentNamespace;
+    private String containerImage = "quay.io/kroxylicious/kroxylicious-developer:" + Environment.KROXY_VERSION;
+    private final ResourceManager resourceManager = ResourceManager.getInstance();
+
+    /**
+     * Instantiates a new Kroxy.
+     *
+     * @param deploymentNamespace the deployment namespace
+     */
+    public Kroxy(String deploymentNamespace) {
+        this.deploymentNamespace = deploymentNamespace;
+        if (!Objects.equals(Environment.QUAY_ORG, Environment.QUAY_ORG_DEFAULT)) {
+            containerImage = "quay.io/" + Environment.QUAY_ORG + "/kroxylicious:" + Environment.KROXY_VERSION;
+        }
+    }
+
+    /**
+     * Deploy - Port per broker plain config
+     * @param testInfo the test info
+     */
+    public void deployPortPerBrokerPlain(TestInfo testInfo, int replicas) {
+        LOGGER.info("Deploy Kroxy in {} namespace", deploymentNamespace);
```

Comment:
I think what you have here is a great start.  Eventually this will be replaced with creation of a Kroxylicious CR(s) rather than the underlying objects.